app/src/main/java/com/sopt/geonppang/util/binding/BindingDialogFragment.kt

```diff
@@ -23,6 +25,17 @@ abstract class BindingDialogFragment<B : ViewDataBinding>(@LayoutRes private val
         return binding.root
     }
 
+    override fun onStart() {
+        super.onStart()
+        dialog?.window?.setLayout(
+            WindowManager.LayoutParams.MATCH_PARENT,
+            WindowManager.LayoutParams.MATCH_PARENT,
+        )
+        dialog?.window?.setBackgroundDrawableResource(R.color.dialog_background)
```

Comment:
선생님 이거 에뮬 새로 만들어서 확인해보니까 이렇게 안 해도 배경 색상 바뀌어서 요렇게 수정해주실 수 있겠습니까??
이슈 새로 파서 하려 했는디 요거 `BindingDialogFragment`에 추가했던 거 기억나서,,,
부탁합니다 ~
그리고 color 리소스에서 `<color name="dialog_background">#80121212</color>` 이 색상도 삭제 부탁이염 ㅠㅠ
여기서만 사용되어서!!


```suggestion
            WindowManager.LayoutParams.WRAP_CONTENT,
        )
        dialog?.window?.setBackgroundDrawableResource(android.R.color.transparent)
```